sdk/eventhub/event-hubs/README.md

```diff
@@ -189,10 +172,14 @@ const partitionIds = await client.getPartitionIds();
 const subscription = client.subscribe(
   EventHubClient.defaultConsumerGroupName,
   (receivedEvents, context) => { },
-  [ partitionIds[0] ], {
+  partitionIds[0], {
     defaultEventPosition: EventPosition.latest()
   }
 );
+
+// events will be fired off as messages are received
+
+subscription.close();
 ```
 
 The [Inspect an Event Hub](#inspect-an-event-hub) example shows how to get the list of partition ids.
```

Comment:
A few lines below there is a line `You can use this consumer in one of 3 ways to receive events:`

That was for the time we had pull, push and aync iterators.
Update this to reflect the 3 overloads to the `subscribe` method

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/2c47d67e3c7402e11860d09c0caea074547dc1be